gulpfile.js

```diff
@@ -29,6 +38,68 @@ gulp.task('templates', function() {
     .pipe(gulp.dest(outputPath + 'js'));
 });
 
+/*********************************************************
+ * JavaScript
+ ********************************************************/
+
+gulp.task('js:build', function() {
```

Comment:
Why do we need the gulp task can we do this with https://github.com/substack/watchify directly from the `npm scripts`?